tests/test_unit.py

```diff
@@ -1899,36 +1899,59 @@ def test_happy_path(self):
                 event = build_event(event_type, host)
                 headers = message_headers(event_type, host_id)
 
-                self.event_producer.write_event(event, host_id, headers)
+                self.event_producer.write_event(event, host_id, headers, wait=True)
```

Comment:
This is testing single-host operations that do not use `wait=True`
```suggestion
                self.event_producer.write_event(event, host_id, headers)
```

Reply:
This suggestion still stands. `wait=True` is used in bulk operations (like our bulk deletion endpoint), but this test is validating single-host operations.